examples/treeview/treeview-1/js/tree.js

```diff
@@ -93,9 +93,11 @@ Tree.prototype.setFocusToItem = function (treeitem) {
 
     if (ti === treeitem) {
       ti.domNode.tabIndex = 0;
+      ti.domNode.ariaSelected = true;
```

Comment:
To set the `aria-selected` attribute, you'd need to use `el.setAttribute('aria-selected', 'true')`. AOM attribute reflection isn't broadly supported, or stable.

The selected state should also likely be set to the actual selected item, and not the currently focused item. We should also probably look into adding a visual selected state, though we could conceivably add the attribute even without the visual state.